### DISCUSSION TITLE
cyipopt.Problem not found

### DISCUSSION THREAD

**Discussion by wc754375721** [coin-or/Ipopt#504] [Q&A] [answered]
When I use cyipopt in Python, an error called module 'cyipopt' has no attribute "Problem" occurs.what should i do to solve this problem?I would be grateful if anyone could help me out.

**Comment by svigerske** [comment, marked as answer]
That should be asked at https://github.com/mechmotum/cyipopt. And they probably would need much more information to be helpful.

**Reply by wc754375721** [reply]
OK,thank you.I will try later
